cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/ProjectController.java

```diff
@@ -0,0 +1,82 @@
+package mil.army.usace.hec.cwms.radar.client.controllers;
+
+import static mil.army.usace.hec.cwms.radar.client.controllers.RadarEndpointConstants.ACCEPT_HEADER_V2;
+import static mil.army.usace.hec.cwms.radar.client.controllers.RadarEndpointConstants.ACCEPT_QUERY_HEADER;
+
+import java.io.IOException;
+import mil.army.usace.hec.cwms.http.client.ApiConnectionInfo;
+import mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl;
+import mil.army.usace.hec.cwms.http.client.HttpRequestResponse;
+import mil.army.usace.hec.cwms.http.client.request.HttpRequestExecutor;
+import mil.army.usace.hec.cwms.radar.client.model.Project;
+import mil.army.usace.hec.cwms.radar.client.model.Projects;
+import mil.army.usace.hec.cwms.radar.client.model.RadarObjectMapper;
+
+
+public final class ProjectController {
+    private static final String PROJECT_ENDPOINT = "projects";
+
+    public Project retrieveProject(ApiConnectionInfo apiConnectionInfo, ProjectEndpointInput.GetOne input)
+            throws IOException {
+        String endpoint = PROJECT_ENDPOINT + "/" + input.projectId();
+        HttpRequestExecutor executor = new HttpRequestBuilderImpl(apiConnectionInfo, endpoint)
+                .addQueryHeader(ACCEPT_QUERY_HEADER, ACCEPT_HEADER_V2)
```

Comment:
Shouldn't these be JSON V1 instead of V2?

Reply:
I'm unsure how to handle this.  The V1 header now has version=1 tacked on to it.   The CDA JSONv1 formatter doesn't do the right thing.  So on the CDA side I'm going to have the class be JSONv2 with a JSON alias and (I think?) ignore application/json;version=1

Reply:
Given this is the first version of the Project data it's official full name would be "application/json;version=1" with an "application/json" alias. The client API at this level probably should be coded against the full name.

Reply:
https://github.com/USACE/cwms-data-api/blob/2fd6d1619595ccf5e51608fbbcad525be3dd3d1f/cwms-data-api/src/main/java/cwms/cda/data/dto/project/Project.java#L45

That should be JSONv1 with an alias to plain JSON. Sorry, we should've caught then during the initial PR. (The formatter class is Correct and we should probably give it a better name since it's more about a natural mapping than being "version 2"

Reply:
https://github.com/USACE/cwms-data-api/pull/766 will fix the CDA side of this